docker-compose.yml

```diff
@@ -23,3 +24,8 @@ volumes:
       type: none
       device: $PWD
       o: bind
+  cache:
+    driver_opts:
+      type: none
+      device: $PWD/.cache
+      o: bind
```

Comment:
Consider whether this could be simplified and optimized.

As I understand it this will effectively mount the `.cache` directory twice: Once in the root because it is included in `$PWD` and once in the node cache.

I get why we want to use a directory within the project as the cache dir but [we could configure Yarn to use `./.cache` as the cache folder using an environment variable for the Node image](https://classic.yarnpkg.com/en/docs/cli/cache/#toc-change-the-cache-path-for-yarn).

This way we would not have to add an extra volume.

Reply:
Fixed